whitepapers/0015_notary.md

```diff
@@ -162,12 +162,24 @@ All three systems:
 - **Action**: Approve or reject based on accounting rules
 - **Focus**: Prevention of erroneous token unlocks, reducing the effectiveness of cross-chain exploits
 
+#### Sequential Processing and Cumulative Delays
+
+Messages flow through the Notary, Governor, and Accountant sequentially. Each system independently evaluates messages and maintains its own delay queue. When a message's delay period expires in one system, it proceeds to the next system for evaluation.
+
+This sequential architecture means delays can accumulate. In the worst case, a single message could be:
+1. Delayed 4 days by the Notary (e.g., for a "Rejected" verification state)
+2. Then delayed 24 hours by the Governor (if it exceeds value thresholds)
+3. Then potentially delayed or rejected by the Accountant (based on accounting rules)
```

Comment:
Does the Accountant delay? I thought it just blocked further processing if violated?

Reply:
Yeah that's all I mean. I'll rephrase here.